packages/ui/src/components/postCollectionViews/SummaryCollectionView.tsx

```diff
@@ -0,0 +1,131 @@
+import { useNavigation } from '@react-navigation/native';
+import * as db from '@tloncorp/shared/db';
+import { forwardRef, useEffect, useMemo } from 'react';
+import { FlatList } from 'react-native';
+import { View, getTokenValue } from 'tamagui';
+
+import { useLivePost } from '../../contexts';
+import { usePostCollectionContextUnsafelyUnwrapped } from '../../contexts/postCollection';
+import { ForwardingProps } from '../../utils/react';
+import { ListItem } from '../ListItem';
+import Pressable from '../Pressable';
+import { IPostCollectionView } from './shared';
+
+export function BaseSummaryCollectionView({
+  items,
+  onPressItem,
+  ...forwardedProps
+}: ForwardingProps<
+  typeof View,
+  {
+    items: db.Post[];
+    onPressItem?: (post: db.Post, index: number) => void;
+  }
+>) {
+  return (
+    <View {...forwardedProps}>
+      <FlatList
+        data={items}
+        renderItem={({ item, index }) => (
+          <PostSummary
+            item={item}
+            onPress={() => {
+              onPressItem?.(item, index);
+            }}
+          />
+        )}
+        keyExtractor={(item) => item.id}
+        contentContainerStyle={{
+          paddingHorizontal: getTokenValue('$space.xl'),
+        }}
+      />
+    </View>
+  );
+}
+
+function PostSummary({
+  item,
+  onPress,
+}: {
+  item: db.Post;
+  onPress?: () => void;
+}) {
+  const post = useLivePost(item);
+
+  const { title, subtitle, authorId } = useMemo(
+    () => ({
+      key: post.id,
+      authorId: post.authorId,
+      ...(post.title
+        ? {
```

Comment:
When testing this against `createFakePosts` creating notes, I see title / subtitle here – not sure how to get that from one of the app inputs. (I thought gallery input had title, but I don't see it anymore...)

Reply:
aha, that's a bug in the custom channels branch then, looks like it always uses the gallery big input